package-builder/flox-build.mk

```diff
@@ -355,8 +359,7 @@ define BUILD_template =
   $(eval _pvarname = $(subst -,_,$(_pname)))
   # Identify result symlink basename.
   $(eval _result = result-$(_pname))
-  # Eventually derive version somehow, but hardcode it in the meantime.
-  $(eval _version = 0.0.0)
+  $(eval _version = $(if $(filter null,$(_version)),unknown,$(_version)))
```

Comment:
should the defaulting be part of the jq expr?

Reply:
I think it's fine how it is.